drivers/builtin/include/mbedtls/crypto_builtin_composites.h

```diff
@@ -20,8 +20,8 @@
  *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
  */
 
-#ifndef PSA_CRYPTO_BUILTIN_COMPOSITES_H
-#define PSA_CRYPTO_BUILTIN_COMPOSITES_H
+#ifndef TF_PSA_CRYPTO_MBEDTLS_CRYPTO_BUILTIN_COMPOSITES_H
```

Comment:
Debatable: since these headers are being moved anyway, should they move to `tf-psa-crypto/*` rather than `mbedtls/*`?